### ISSUE TITLE
AAPT: Attribute already defined with incompatible format

### ISSUE THREAD

**Issue by rambabusaravanan** [bertrandmartel/youtubetv#1]
I am getting this error while including just the dependency

```
androidtv/tv/build/intermediates/res/merged/debug/values/values.xml:546: AAPT: Attribute "borderWidth" already defined with incompatible format.
androidtv/tv/build/intermediates/res/merged/debug/values/values.xml:436: AAPT: Original attribute defined here.
androidtv/tv/build/intermediates/res/merged/debug/values/values.xml:546: error: Attribute "borderWidth" already defined with incompatible format.
androidtv/tv/build/intermediates/res/merged/debug/values/values.xml:436: Original attribute defined here.
```

**Comment by bertrandmartel** [comment]
Thank you for noticing this, I will add namespace prefix for all attributes to fix this issue

**Comment by bertrandmartel** [comment]
I've just released v1.2 with namespace prefix for all attributes : 
```
compile 'fr.bmartel:youtubetv:1.2'
```
It's already live on JCenter & soon available on MavenCentral

For instance, with the new attrs :
```
<fr.bmartel.youtubetv.YoutubeTvView
    android:id="@+id/youtube_video"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    app:yt_videoId="4a5m4qF1e6Q" />
```
